cli/types/tests/kitchen-sink.ts

```diff
@@ -116,3 +116,5 @@ const obj = {
   foo: () => { }
 }
 cy.spy(obj, 'foo').as('my-spy')
+
+cy.window().then(window => window.eval('1'))
```

Comment:
It would be nice to have a type assertion nailing down exactly what type we expect the compiler to infer for `window`

Reply:
Sure I can add one. You might want to consider using dtslint for that (`$ExpectType`), because annotations could result in false negatives:

```ts
declare const anything: any;
// no error!!!!
const myWindow: Window = anything;
```

I won't do that for this PR though, as that requires changing infrastructure.

Reply:
`dtslint` is actually integrated! You could add `dtslint` assertions here for instance: https://github.com/cypress-io/cypress/blob/develop/cli/types/tests/cypress-tests.ts

Reply:
Done!